filebeat/module/traefik/access/ingest/pipeline.json

```diff
@@ -50,13 +50,22 @@
         {
             "rename": {
                 "field": "traefik.access.agent",
-                "target_field": "traefik.access.user_agent.original"
+                "target_field": "user_agent.original"
             }
         },
         {
-            "geoip": {
+            "grok": {
                 "field": "traefik.access.remote_ip",
-                "target_field": "traefik.access.geoip"
+                "patterns": [
+                    "^(%{IP:source_ecs.ip}|%{HOSTNAME:source_ecs.domain})$"
```

Comment:
`source` is gone in the Filebeat code, we can map directly to `source.domain`

Reply:
Again, waiting for #8983 to be merged.